addons/sourcemod/scripting/include/surftimer.inc

```diff
@@ -151,15 +168,15 @@ native bool surftimer_SafeTeleport(int client, float fDestination[3], float fAng
  * @param client			The client's ID.
  * @return						Plugin_Handled, to stop the client from searching.
  */
-forward Action surftimer_OnMapFinished(int client, float fRunTime, char sRunTime[54], int rank, int total, int style);
+forward Action surftimer_OnMapFinished(int client, float fRunTime, char sRunTime[54], float PBDiff, float WRDiff, int rank, int total, int style);
```

Comment:
What's the reason for this change? Plugins which use the old version, will not work anymore.

Reply:
this was intented to proved the time differences of the player.
i added it to this forward since it made sense for me to have the variables in the forward that gets called when the player finishes a run